cmd/common/trace/fsslower.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"fmt"
+	"strings"
+
+	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/fsslower/types"
+	"github.com/spf13/cobra"
+)
+
+type FsSlowerFlags struct {
+	FsSlowerMinLatency uint
+	FsSlowerFilesystem string
+}
```

Comment:
I know it was already like that, but could you please remove the `FsSlower` prefix, it is redudant:

```suggestion
type FsSlowerFlags struct {
	MinLatency uint
	Filesystem string
}
```